app/src/main/res/layout/activity_add_deadline.xml

```diff
@@ -176,10 +170,21 @@
             android:onClick="selectNotifications"
             android:text="@string/select_notifications"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.515"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/constraintPDFSelection" />
 
+        <Button
+            android:id="@+id/add_deadline_select_group"
+            android:layout_width="wrap_content"
+            android:layout_height="49dp"
```

Comment:
can you not use wrap_content here ? Or it's nicer with these parameter ?

Reply:
I think this is good but I can change it without any problem